www/class/centreonWidget.class.php

```diff
@@ -224,21 +224,23 @@ public function getWidgetInfo($type = "id", $param)
     /**
      * Add widget to view
      *
-     * @param array $params
+     * @param int $customViewId
+     * @param int $widgetModelId
+     * @param string $widgetTitle
      * @throws CentreonWidgetException
      */
-    public function addWidget($params)
+    public function addWidget($customViewId, $widgetModelId, $widgetTitle)
     {
-        if (!isset($params['custom_view_id'])
-            || !isset($params['widget_model_id'])
-            || !isset($params['widget_title'])
+        if (!isset($customViewId)
+            || !isset($widgetModelId)
+            || !isset($widgetTitle)
```

Comment:
isset can take multiple argument 
```suggestion
        if (!isset($customViewId, $widgetModelId, $widgetTitle)
```